lib/igniter/code/function.ex

```diff
@@ -367,6 +371,23 @@ defmodule Igniter.Code.Function do
     end
   end
 
+  @doc """
+  Checks if the provided function call (in a Zipper) has an argument that equals
+  `term` at `index`.
+  """
+  @spec argument_equals?(Zipper.t(), integer(), any()) :: boolean()
+  def argument_equals?(zipper, index, term) do
+    if function_call?(zipper) do
+      Igniter.Code.Function.argument_matches_predicate?(
+        zipper,
+        index,
+        &Igniter.Code.Common.nodes_equal?(&1, term)
+      )
+    else
+      raise "The provided zipper is not a function call."
```

Comment:
Let's just return `false` here, and leave it up to them to validate that they are in a function call.
```suggestion
      false
```

Reply:
Sure, that works. Just didn't want to return an `:error` (as it was, probably mistakenly, returned in `configures?/3` before) as it introduces a subtle bug since it's truthy.